include/experimental/kernel/gemm/impl/int4_dequantize_kslicing_xe.hpp

```diff
@@ -451,9 +451,8 @@ class gemm_universal_t<
   static cl::sycl::range<3> get_local_range() {
     uint32_t local_range_m = (wg_tile_m + sg_tile_m - 1) / sg_tile_m;
     uint32_t local_range_n = (wg_tile_n + sg_tile_n - 1) / sg_tile_n;
-    // std::cout << "Local range: {" << num_local_kslicing << ", " <<
-    // local_range_m
-    //           << ", " << local_range_n << "} \n";
+    std::cout << "Local range: {" << num_local_kslicing << ", " << local_range_m
+              << ", " << local_range_n << "} \n";
```

Comment:
Add macro to disable it in IPEX (preferably disable by default and enable in NS).

Reply:
Use XETLA_PRINTF to replace cout.
”cmake -DLOG=on“ to turn on. The default is off.

Reply:
https://github.com/intel/neural-speed/pull/176/commits/d973255b421e940f2634abb8652e73b28c2a3988